src/storage/btree.c

```diff
@@ -14804,14 +14804,14 @@ btree_coerce_key (DB_VALUE * keyp, int keysize, TP_DOMAIN * btree_domainp, int k
 			}
 		      else
 			{	/* CASE 4 */
-			  ;	/* nop */
+			  minmax = BTREE_COERCE_KEY_WITH_MAX_VALUE;
 			}
 		    }
 		}
 
 	      if (minmax == BTREE_COERCE_KEY_WITH_MIN_VALUE)
 		{
-		  if (dsize < keysize)
+		  if (dsize <= keysize)
```

Comment:
What if dsize > keysize?

Reply:
I don't think there's such case: (dsize > keysize).
Do you think it is possible?

I'm not sure dsize==keysize case also need to set min_max_val.
If the case also needs, do you think we need `if`?

Reply:
dsize == keysize needs min_max_val. ```if``` should be removed.

Reply:
yes, indeed :)